.github/workflows/inductor.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           { config: "inductor_timm_dynamic", shard: 1, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
           { config: "inductor_timm_dynamic", shard: 2, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
           { config: "inductor_torchbench_dynamic", shard: 1, num_shards: 1, runner: "linux.g5.4xlarge.nvidia.gpu" },
-          { config: "inductor_distributed", shard: 1, num_shards: 1, runner: "linux.g5.12xlarge.nvidia.gpu" },
+          { config: "inductor_distributed", shard: 2, num_shards: 2, runner: "linux.g5.12xlarge.nvidia.gpu" },
```

Comment:
Why do we need two shards here? Are we testing distributed model across nodes?

Reply:
ah i thought this was the distributed shards i.e. gpu numbers. the job does still have access to all local gpus even when i set this to 1